LoginPage.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const response = await fetch('/api/login', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ email, password })
+    });
+
+    const data = await response.json();
+
```

Comment:
There is no user feedback if the login attempt is unsuccessful, apart from the URL not changing. Consider adding a user-friendly error message.